### PR TITLE
Update developer guide for matchBuyer acitivity diagram

### PR DESCRIPTION
**Pull request by felixchanyy**
To summarise matchBuyer execution.

### CODE REVIEW

File: docs/diagrams/MatchBuyerActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+else([parameter is valid])
+    :Create FullNameEqualsKeywordPredicate with keyword;
+    :MatchBuyerCommand is executed;
+    :Model update filtered seller list with predicate;
+    :Display list of sellers with matching houses;
+endif
+
+stop
+
+@enduml
```
Comment (lokidoki102) [inline docs/diagrams/MatchBuyerActivityDiagram.puml:20]: missing EOF

**Review by lokidoki102 — APPROVED** [review]

File: docs/diagrams/MatchBuyerActivityDiagram.puml (hunk 1 above)
Comment (felixchanyy) [reply docs/diagrams/MatchBuyerActivityDiagram.puml]: Thanks. Fixed

**Review by lokidoki102 — APPROVED** [review]
LGTM!

**Review by KhoonSun47 — APPROVED** [review]
LGTM! Good Job!

**Pull request merged**